.github/actions/restore-node/action.yml

```diff
@@ -132,8 +132,9 @@ runs:
       working-directory: ${{ inputs.path }}
       run: |-
         set -x
-        # In case of Endo override, ignore staged changes.
-        if [ -n "$(git status --porcelain | grep -Eve '^A  '; true)" ]; then
+        # In case of Endo override, ignore matching index and worktree.
+        # (First column is non-space, second column is space, followed by separator.)
+        if [ -n "$(git status --porcelain | grep -Eve '^[^ ]  '; true)" ]; then
```

Comment:
I think `'^M  '` would work too, right?

Reply:
Yes, that should work too.  Would you prefer it?

Reply:
It'd be a little clearer, but I don't care enough to request a change